mgmt/WebMgmtUtils.cc

```diff
@@ -226,16 +236,21 @@ varSetInt(const char *varName, RecInt value, bool convert)
   case RECD_INT:
     RecSetRecordInt((char *)varName, (RecInt)value, REC_SOURCE_EXPLICIT);
     break;
+
   case RECD_COUNTER:
     if (convert) {
       RecSetRecordCounter((char *)varName, (RecCounter)value, REC_SOURCE_EXPLICIT);
       break;
     }
+  // fallthrough
+
```

Comment:
This all seems odd an inconsistent. You positive this is in fact not finding a bug in the missing breaks?